pkgs/by-name/am/amber-lang/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+let
+  version = "0.3.1-alpha";
+in
+rustPlatform.buildRustPackage {
+  pname = "amber-lang";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "Ph0enixKM";
+    repo = "Amber";
+    rev = version;
+    hash = "sha256-VSlLPgoi+KPnUQJEb6m0VZQVs1zkxEnfqs3fAp8m1o4=";
+  };
+
+  cargoHash = "sha256-NzcyX/1yeFcI80pNxx/OTkaI82qyQFJW8U0vPbqSU7g=";
+
+  doCheck = false;
```

Comment:
Tests not passing, hopefully a new release soon (last was 7 months ago) that fixes it.